src/instructlab/cli/process/list.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.process.process import list_processes
+from instructlab.utils import print_table
+
+
+@click.command(name="list")
+@clickext.display_params
+def list():
+    process_list = list_processes()
+    if process_list is not None:
+        print_table(["Type", "PID", "UUID", "Log File", "Runtime"], process_list)
```

Comment:
Seems it would better to add the `start time` , `state`, `command` in the `list`, specially what commands it run and what parameters it used,  can simply to check, that might help to troubleshoot.
like the common process list `ps`:
```
]# ps aux
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root           1  0.0  0.0 242836 15992 ?        Ss   Dec03   1:35 /usr/lib/systemd/systemd --system
```